### DISCUSSION TITLE
Fastfetch not displaying logo during update

### DISCUSSION THREAD

**Discussion by sanjyay** [prasanthrangan/hyprdots#1492] [Q&A] [answered]
I generated fastfetch config files using fastfetch --gen-config
![image](https://github.com/prasanthrangan/hyprdots/assets/102979855/99d7b43e-3622-479b-8415-09b8df557fd5)

And i also copied pngs directory from ~/hyprdots/Configs/.config/fastfetch.

Now this is my output when i type fastfetch in the terminal 
![image](https://github.com/prasanthrangan/hyprdots/assets/102979855/4dd54ede-38cb-4b91-8814-d54c7fe25ff3)

and this is my output when i click the updates button
![image](https://github.com/prasanthrangan/hyprdots/assets/102979855/b763eaf9-d1eb-42bd-8932-a7f1daabba62)

**Comment by ghost** [comment]
This was fixed in commit https://github.com/prasanthrangan/hyprdots/commit/7699077ea51f3f7a63abbdf09fb989ec61390a57 13 days ago, please update your configs.

**Reply by sanjyay** [reply]
I update my configs today before this thing happened. Infact if i delete my ~/.configs/fastfetch then this is the output when I click updates
![image](https://github.com/prasanthrangan/hyprdots/assets/102979855/d1a7bc9b-a9e4-452b-9126-912c374de785)
the png images don't appear

**Reply by ghost** [reply, marked as answer]
> I update my configs today before this thing happened.

Can you run `cat ~/.config/waybar/modules/updates.jsonc` just so i can see what the file looks like for you?

> Infact if i delete my ~/.configs/fastfetch then this is the output when I click updates

Yes, you're now using the default config, and it's displaying ascii art, which is just text and is displayable anywhere while a png can't.

**Reply by sanjyay** [reply]
I had only performed git pull . I saw the changes within Hyprdots directory but it wasn't updated here ~/.config/waybar/modules/updates.jsonc. Thank you

**Reply by F4LC0N69** [reply]
Install imagemagick and w3m once. I just solved that issue randomly.

**Comment by Adityak102006** [comment]
can you give me your fastfetch config file

**Reply by sanjyay** [reply]
i am using hyprdots' fastfetch config only

**Reply by Adityak102006** [reply]
Okk

**Comment by Y0usaf3** [comment]
u just leaked ur ip adress @sanjyay  delet the image before it's too late

**Reply by switchmaxfx** [reply]
2025 and people still dont know the difference between a local IP and a public IP.
